packages/kbn-test/src/kbn_client/kbn_client_requester.ts

```diff
@@ -13,6 +13,7 @@ import Qs from 'querystring';
 import Axios, { AxiosResponse, ResponseType } from 'axios';
 import { isAxiosRequestError, isAxiosResponseError } from '@kbn/dev-utils';
 import { ToolingLog } from '@kbn/tooling-log';
+import { cleanException } from '../auth/saml_auth';
```

Comment:
__question__: since `cleanException` is no longer a `saml_auth`-only helper function, maybe we better place it somewhere upper level like `src/clean_exception.ts`

Reply:
Not a bad idea at all!